perplexity.go

```diff
@@ -107,12 +112,21 @@ type Client struct {
 }
 
 // NewClient creates a new Perplexity API client
-func NewClient(apiKey, model string) *Client {
+func NewClient(apiKey, model string, options ...*ClientOptions) *Client {
+	var opts ClientOptions
+	if len(options) > 0 {
+		opts = *options[0]
+	} else {
+		opts = ClientOptions{
+			RequestTimeoutInSeconds: 30,
+		}
+	}
+
```

Comment:
I think it could be better to have only one ClientOptions and if it's nil then apply the defaults. There no need to have a slice of ClientOpions cuz it will have everything in a single struct.

Reply:
Okay, let's simplify it a little bit. I will be a breaking change though